inspirehep/modules/records/jsonschemas/records/jobs.json

```diff
@@ -22,21 +22,17 @@
             "uniqueItems": true
         },
         "continent": {
-            "items": {
-                "enum": [
-                    "North America",
-                    "Europe",
-                    "Asia",
-                    "South America",
-                    "Africa",
-                    "Oceania"
-                ],
-                "title": "Continent",
-                "type": "string"
-            },
-            "title": "Continent",
-            "type": "array",
-            "uniqueItems": true
+            "type": "string",
+            "enum": [
+                "North America",
+                "Europe",
+                "Asia",
+                "South America",
+                "Australasia",
+                "Middle East",
+                "Africa",
+                "Europe / South America"
```

Comment:
That's actually my argument for doing it manually — it's just a handful of them anyway.